### ISSUE TITLE
[a11y][Violation][2.4.1 Bypass Blocks] Multiple elements with "toolbar" roles do not have unique labels

### ISSUE THREAD

**Issue by nkn2022** [jupyter/notebook#7482]
## Description

I am running an accessibility test suite with Notebook UI using the **IBM Equal Access Accessibility Checker** tool on Chrome browser. The results look good overall, only a few things are missing to reach a first formal level of compliance. So kindly help look into these. 

Issues reported were violating **WCAG 2.2** requirements. 
Ref: https://www.ibm.com/able/requirements/checker-rule-sets

Full report (includes information on element locations where issue were found): 
[Accessibility_Report-7a73e10e-f8a4-4a06-a78d-5174a1309c5b.xlsx](https://github.com/user-attachments/files/17400350/Accessibility_Report-7a73e10e-f8a4-4a06-a78d-5174a1309c5b.xlsx)

# Category: `2.4.1 Bypass Blocks`
Ref: https://www.ibm.com/able/requirements/requirements/?version=v7_3#2_4_1

****Violation of Rule:** `aria_toolbar_label_unique`**

> List of issues with elements that violate this rule with a screenshots
> > **Issue 1** 
> > Multiple elements with "toolbar" roles do not have unique labels
> > **Element location:** 
> > `<jp-toolbar style="min-height: var(--jp-private-toolbar-height);" role="toolbar" orientation="horizontal" aria-orientation="horizontal" class="lm-Widget jp-Toolbar jp-cell-menu jp-cell-toolbar">`
> > **Screenshot:**
> >![Image](https://github.com/user-attachments/assets/f5a8327b-13ff-44de-a954-695be459035a)

## Reproduce

- Install Chrome extension for IBM accessibility checker tool. Ref: https://www.ibm.com/able/toolkit/verify/automated
- Open Notebook UI on Chrome browser. Add some cells with code.
- Run the tool and we can see the above `Violation`s reported.

Here is the screenshot of the UI scanned for the attached report:
![Image](https://github.com/user-attachments/assets/895eebec-51ad-43cd-93bb-99c4a8193b68)

## Expected behavior

No violations are reported on the scanning to reach a first formal level of compliance.

**Comment by nkn2022** [comment]
@RRosio Of course. Happy to help. I will open separate ones with screenshots.

**Comment by nkn2022** [comment]
@RRosio I have created one based on a category of the issue reported with details. Kindly let me know if this breakdown looks good.
`4.1.2 Name, Role, Value` - https://github.com/jupyter/notebook/issues/7488

**Comment by nkn2022** [comment]
@RRosio I have also provided information on the tool I used to test and it is pretty easy to set up and run on a Chrome browser. This can be used to confirm that the issue is resolved after the fix. Thanks!

**Comment by nkn2022** [comment]
Reference on what to do for this kind of issue: https://www.ibm.com/able/toolkit/develop/text-and-non-text/#aria-roles
